Source/WebCore/platform/graphics/gstreamer/GStreamerCommon.cpp

```diff
@@ -570,17 +570,15 @@ void registerWebKitGStreamerElements()
         if (auto factory = adoptGRef(gst_element_factory_find("isofmp4mux")))
             gst_plugin_feature_set_rank(GST_PLUGIN_FEATURE_CAST(factory.get()), GST_RANK_PRIMARY + 1);
 
-        WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN // GLib port
         // The VAAPI plugin is not much maintained anymore and prone to rendering issues. In the
         // mid-term we will leverage the new stateless VA decoders. Disable the legacy plugin,
         // unless the WEBKIT_GST_ENABLE_LEGACY_VAAPI environment variable is set to 1.
-        const char* enableLegacyVAAPIPlugin = getenv("WEBKIT_GST_ENABLE_LEGACY_VAAPI");
-        if (!enableLegacyVAAPIPlugin || !strcmp(enableLegacyVAAPIPlugin, "0")) {
+        auto enableLegacyVAAPIPlugin = StringView::fromLatin1(g_getenv("WEBKIT_GST_ENABLE_LEGACY_VAAPI"));
```

Comment:
For new code, even when using StringView is perfectly ok, I think CStringView is a lighter class. No strong opinion on this though.